File: 26_where_do_i_belong.js
// where do I belong

function getIndexToIns(arr, num){
    arr.sort(function(a, b){
        return a-b;
    });

    for (var i=0; i<arr.length; i++){
        if (num <= arr[i]){
            return i;
        }
    }

    return arr.length;
}


console.log(getIndexToIns([40,60,5],50));
console.log(getIndexToIns([2,5,10],15))

File: 26_where_do_i_belong.py
# where do I belong

def getIndexToIns(arr, num):

    for i in range(len(sorted(arr))):
        if num <= arr[i]:
            return i+1
    return len(arr)



def getIndexToInsAnother(arr, num):
    x = arr
    arr.append(num)
    return sorted(x).index(num)

print(getIndexToIns([40,60,5],50))
print(getIndexToIns([2,5,10],15))
print(getIndexToInsAnother([40,60,5],50))
print(getIndexToInsAnother([2,5,10],15))